### DISCUSSION TITLE
升级V5后，原来的Less文件中使用的变量，在新版中怎么用

### DISCUSSION THREAD

**Discussion by xclw2000** [ant-design/ant-design#49187] [General]
### What problem does this feature solve?

以前在Less中可以使用内置的颜色变量, 比如@primary-1,.....@primary-7
现在怎么使用？

### What does the proposed API look like?

无

**Comment by afc163** [comment]
https://ant.design/docs/react/migration-v5-cn#less-%E8%BF%81%E7%A7%BB

**Reply by xclw2000** [reply]
我的意思是在css-in-js里，如何通过token取到这些变量
```
const useStyles = createStyles(({ token, css }) => {
  return {
    title: {
      fontSize: 30,
      color: token.primary6,
      '> img': css`
        width: 32px;
        height: 32px;
        cursor: pointer;
      `,
    },
  };
});
```

token中没有类似primary1,....,primary9这样的属性
